src/btree/bt_handle.c

```diff
@@ -1107,7 +1107,9 @@ __btree_page_sizes(WT_SESSION_IMPL *session)
 #define WT_MIN_PAGES 10
     if (!F_ISSET_ATOMIC_32(conn, WT_CONN_CACHE_POOL) && (cache_size = conn->cache_size) > 0)
         btree->maxmempage = (uint64_t)WT_MIN(btree->maxmempage,
-          ((conn->evict->eviction_dirty_trigger * cache_size) / 100) / WT_MIN_PAGES);
+          ((__wt_atomic_load_double_relaxed(&conn->evict->eviction_dirty_trigger) * cache_size) /
+            100) /
+            WT_MIN_PAGES);
```

Comment:
np: can you save it to a local variable

```suggestion
eviction_dirty_trigger = __wt_atomic_load_double_relaxed(&conn->evict->eviction_dirty_trigger);
btree->maxmempage = (uint64_t)WT_MIN(btree->maxmempage, 
                 ((eviction_dirty_trigger * cache_size) / 100) / WT_MIN_PAGES);
```

Reply:
updated.